run-tests.sh

```diff
@@ -0,0 +1,163 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+cd "$(dirname "$0")"
+
+generate_id() {
+  LC_ALL=C tr -dc 'a-z' </dev/urandom | head -c 13 ; echo
```

Comment:
Why the `; echo`?

Reply:
for newline

Reply:
Why do you need a newline?

Reply:
I guess in this case it's not necessary.